config/packages/nelmio_security.yaml

```diff
@@ -31,7 +31,7 @@ nelmio_security:
             base-uri: '%demosplan_csp_base_uri%'
             form-action: '%demosplan_csp_form_action%'
             worker-src:
-                - 'none'
+                - 'blob:'
```

Comment:
This change seems to be unrelated to the current PR.

Reply:
actually no. Bumping the masterPortalApi made this change necessary. So its the last snippet of the change-reaction while "you not simply bump node"